lib/agent0/agent0/hyperdrive/exec/trade_loop.py

```diff
@@ -69,14 +76,28 @@ def trade_if_new_block(
                     float(trade_result.trade_object.market_action.trade_amount),
                 )
             elif trade_result.status == TradeStatus.FAIL:
-                logging.info(
-                    "AGENT %s (%s) attempted %s for %g\nCrashed with error: %s",
-                    str(trade_result.agent.checksum_address),
-                    trade_result.agent.policy.__class__.__name__,
-                    trade_result.trade_object.market_action.action_type,
-                    float(trade_result.trade_object.market_action.trade_amount),
-                    trade_result.exception,
+                is_slippage = isinstance(trade_result.exception, ContractCustomError) and (
```

Comment:
did you confirm with Alex that this is always bc of slippage?

Reply:
Confirmed